.github/workflows/build-and-test_release.yml

```
@@ -1462,23 +1462,15 @@ jobs:
       # </template: login_rw_registry_step>
       - name: Run tests
         env:
-          TESTS_IMAGE_NAME: ${{needs.build_fe.outputs.tests_image_name}}
+          TESTS_IMAGE_NAME: "registry.deckhouse.io/base_images/golang:1.16.3-alpine3.12@sha256:371dc6bf7e0c7ce112a29341b000c40d840aef1dbb4fdcb3ae5c0597e28f3061"
```

```suggestion
          TESTS_IMAGE_NAME: "registry.deckhouse.io/base_images/golang:1.16.3-alpine3.12@sha256:371dc6bf7e0c7ce112a29341b000c40d840aef1dbb4fdcb3ae5c0597e28f3061"
```
I suppose this one should be reverted.

---

i do not understand what you mean. In matrix tests we use alpine go image, not builded deckhouse image.

---

Previously, there was a variable. Now the image is hardcoded.

---

changed to render from image_versions.yaml